src/dscanner/main.d

```diff
@@ -91,6 +92,7 @@ else
 				"declaration|d", &symbolName,
 				"config", &configLocation,
 				"report", &report,
+				"reportFormat", &reportFormat,
```

Comment:
setting this should implicitly set `report` to `true`.

Reply:
yes, really make sense. fixed